File: app/src/main/java/com/sola/anime/ai/generator/domain/model/config/Iap.java
package com.sola.anime.ai.generator.domain.model.config;

import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

@Keep
public class Iap {

    @SerializedName("preview1")
    @Expose
    public List<Preview1> preview1;
    @SerializedName("preview2")
    @Expose
    public List<Preview2> preview2;
    @SerializedName("preview3")
    @Expose
    public List<Preview3> preview3;

}

File: app/src/main/java/com/sola/anime/ai/generator/domain/model/config/iap/IAP.kt
package com.sola.anime.ai.generator.domain.model.config.iap

import androidx.annotation.Keep
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Keep
@Entity(tableName = "IAPS")
class IAP{
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0
    @SerializedName("preview")
    @Expose
    var preview: String = ""
    @SerializedName("ratio")
    @Expose
    var ratio: String = "1:1"
}